ui/src/components/popovers/PasteModal.tsx

```diff
@@ -37,27 +56,40 @@ export default function PasteModal() {
     setPasteMode(false);
     setDisableVideoFocusTrap(false);
     if (rpcDataChannel?.readyState !== "open" || !TextAreaRef.current) return;
+    if (!keyboardLayout) return;
+    if (!chars[keyboardLayout]) return;
 
```

Comment:
Let's assign the lookup and use that lower down like:
```typescript
    if (!keyboardLayout) return;
   const layoutChar = chars[keyboardLayout];
    if (!layoutChars) return;
```

Reply:
Sure, could you send a PR?